pkg/controller/certificates/internal/secretsmanager/secret_test.go

```diff
@@ -71,6 +75,16 @@ func TestSecretsManager(t *testing.T) {
 			"template": "label",
 		}),
 	)
+	// enable feature gate additional private key for this test
+	utilfeature.EnableFeatureGate(feature.ExperimentalCertificatePrivateKeyAdditionalFormats)
```

Comment:
Please use `defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.<FeatureName>, <value>)()` so we can ensure this is *unset* again after the test

Reply:
done :heavy_check_mark: